test/tests/functional/pbs_preemption.py

```diff
@@ -57,9 +57,10 @@ def setUp(self):
         if len(self.moms) == 2:
             self.mom1 = self.moms.keys()[0]
             self.mom2 = self.moms.keys()[1]
-            # Since some tests need multi-node setup and majority don't
-            # deleting the second node for majority, tests which need
-            # multi-node setup will create the second node itself.
+            # Since some tests need multi-node setup and majority don't,
+            # Delete the second node so that single node tests don't fail.
```

Comment:
The previous line ended with a comma.  This is in the middle of the sentence.  Lowercase 'Delete'